tests/test_pyproject.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pathlib
+
+from rsconnect.pyproject import lookup_metadata_file, parse_pyproject_python_requires
+
+import pytest
+
+HERE = os.path.dirname(__file__)
+PROJECTS_DIRECTORY = os.path.abspath(os.path.join(HERE, "testdata", "python-project"))
+
+
+@pytest.mark.parametrize(
```

Comment:
It's a nit, but adding a simple comment here that you are testing to make sure that tests against the sample directories return the expected files would be great.